Assignment/solution_student/index.html

```diff
@@ -15,17 +15,69 @@
+            <li><a href="#">Team</a></li>
+            <li><a href="#">Contact Us</a></li>
+            <li><a href="#">Blog</a></li>
+          </ul>
+        </nav>
+      </div>
 
-      <main>
-        <p>Our HTML starter template!</p>
-      </main>
+      <div id="main">
+        <article>
+          <header>
+            <h1>How I implemented Relaxr in 2 weeks and changed my life.</h1>
+          </header>
+            <img src="images/blog_photo1.jpg">
+             <p><span class="firstcharacter">L</span>orem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+          <footer>
+            Read more >
+          </footer>
+          <hr>
+          <header>
+            <h1>I traveled across the globe thanks to Relaxr’s Excel automation</h1>
+          </header>
+            <img src="images/blog_photo2.jpg">
+             <p><span class="firstcharacter">L</span>orem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+          <footer>
+            Read more >
+          </footer>
+          <hr>
+        </article>
+        <div id="sidebar">
+          <h1>Categories</h1>
+            <ul>
+             <li><a href="#">Success Stories</a></li>
+             <li><a href="#">Stats</a></li>
+             <li><a href="#">How tos</a></li>
+             <li><a href="#">Best of</a></li>
+              <li><a href="#">Business Tips</a></li>
+            </ul>
+          <h1>About Relaxr</h1>
+            <p>quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
 
-      <footer>
-        <p>&copy; FEWD6 & Friends, INC.</p>
-      </footer>
+          <a href="#" id="learn">Learn More</a>
+        </div>
+        <div class="ad">
+          Ad Unit
+        </div>
+      </div>
+     
+      <div id="signup">
+         <a href="#" class="button2">Sign Up Now!</a>
+      </div>
+        
+      <div id="footer">
+        <h4>Relaxr</h4>
+        <a href="#"><img src="images/twitter.png" class="button3"></a>
+        <a href="#"><img src="images/facebook.png" class="button3"></a>
+        <p>Copyright 2015. Relaxr</p>
```

Comment:
This copyright paragraph is appearing off-center (http://d.pr/i/1lMDp) because of a declaration you made on [line 138](https://github.com/GA-Chicago-FEWD6/Week_03_Layout/pull/1/files#diff-755a8c2dc5177dc883c89b12cacba160R138) of the CSS file that includes all `p` tags. So you'll either need to override that `padding-right: 25%` specifically for `footer p`, or change how you make that declaration, which I think is what you want to do (see my comment on [line 138](https://github.com/GA-Chicago-FEWD6/Week_03_Layout/pull/1/files#diff-755a8c2dc5177dc883c89b12cacba160R138))